media-libs/netpbm/netpbm-10.88.01.recipe

```diff
@@ -606,11 +626,16 @@ REQUIRES="
 	lib:libpng$secondaryArchSuffix
 	lib:libtiff$secondaryArchSuffix
 	lib:libz$secondaryArchSuffix
+	lib:libxml2$secondaryArchSuffix
+	devel:libxml2$secondaryArchSuffix
+	cmd:xml2_config
+	lib:libx11$secondaryArchSuffix
 	"
 
 PROVIDES_devel="
 	netpbm${secondaryArchSuffix}_devel = $portVersion compat >= 10
-	devel:libnetpbm$secondaryArchSuffix = 11.68 compat >= 11
+	libnetpbm$secondaryArchSuffix = 11.68 compat >= 11
+	devel:libnetpbm
```

Comment:
Not needed:
```diff
-	libnetpbm$secondaryArchSuffix = 11.68 compat >= 11
-	devel:libnetpbm
```

Reply:
removing `devel:libnetpbm` actually breaks the build for me

Reply:
My bad on that one :)
Should be:
```diff
-	devel:libnetpbm
+	devel:libnetpbm$secondaryArchSuffix = 11.88 compat >= 11
```